detectable/src/main/java/com/blackduck/integration/detectable/python/util/PythonDependencyTransformer.java

```diff
@@ -21,6 +21,9 @@ public List<PythonDependency> transform(File requirementsFile) throws IOExceptio
         List<PythonDependency> dependencies = new LinkedList<>();
         try (BufferedReader bufferedReader = new BufferedReader(new FileReader(requirementsFile))) {
             for (String line; (line = bufferedReader.readLine()) != null; ) {
+                if (line.contains("\u0000")) {
+                    line = line.replaceAll("\u0000", "");
+                }
```

Comment:
Looks good. I'd request to add this in the `formatLine(String line)` method in this class though as it is already handling similar other line formatting tasks.

Reply:
Done